Dependencies/napi/napi-jsi/source/jsi.cpp

```diff
@@ -0,0 +1,484 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```

Comment:
Why is this in source control? How do we make sure it is in sync? If we must put this in source control, we must also add this to the NOTICES file at the root.

Reply:
Actually, now that I think about it. We should add JSI to the notices regardless. We are still pulling it down.